app/oracle.py

```diff
@@ -18,6 +18,24 @@
     level=logging.INFO, format='%(levelname)8s %(asctime)s <daemon> %(message)s', datefmt='%Y-%m-%d %H:%M:%S'
 )
 
+#### print version ####
```

Comment:
The statements looks pretty expressive and self-descriptive, so comments are unneccessary.